src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+  },
+  'ngcc postinstall': {
+    action:
+      'Nếu bạn phụ thuộc vào nhiều thư viện thứ 3 khác nhau, bạn nên xem xét việc thực thi lệnh `ngcc` (Angular Compatibility Compiler) để tăng tốc cho build của dự án. Bạn có thể thực thi `ngcc` dưới dạng `postinstall` trong `package.json` bằng [1 sự thay đổi nhỏ](https://v9.angular.io/guide/ivy#speeding-up-ngcc-compilation).'
+  },
+  'express-universal-server': {
+    action:
+      'Nếu bạn sử dụng Angular Universal với `@nguniversal/express-engine` hoặc `@nguniversal/hapi-engine`, một số backup files sẽ được khởi tạo. Một trong số đó là `server.ts`. Nếu như file của bạn khác với file được khởi tạo mặc định, có thể bạn sẽ phải sao chép một số thay đổi từ `server.ts.bak` vào trong file `server.ts` bằng tay.'
+  },
+  'ivy i18n': {
+    action:
+      'Angular 9 giới thiệu hàm toàn cục (global) `$localize()` và hàm này cần phải được load nếu như ứng dụng của bạn phụ thuộc vào bộ Toàn cầu hoá (i18n) của Angular. Thực thi lệnh `ng add @angular/localize` để thêm vào các thư viện cũng như các thay đổi về code cần thiết. Tham khảo [$localize Global Import Migration guide](https://v9.angular.io/guide/migration-localize) để nắm thêm thông tin chi tiết về sự thay đổi này.'
+  },
+  entryComponents: {
+    action:
+      'Nếu như bạn có khai báo bất kỳ 1 `entryComponents` nào trong bất kỳ 1 `NgModules` nào của bạn, hoặc bạn có bất kỳ chỗ nào sử dụng `ANALYZE_FOR_ENTRY_COMPONENTS`, bạn có thể xoá chúng đi. `entryComponents` không cần thiết nữa với trình biên dịch và môi trường chạy code Ivy'
+  },
+  'testbed-get': {
+    action:
+      'Nếu bạn sử dụng `TestBed.get`, bạn nên chuyển sang dùng `TestBed.inject`. Hàm mới này vẫn giữ hành vi như cũ, nhưng sẽ an toàn về kiểu dữ liệu hơn.'
+  },
+  $localize: {
+    action:
+      'Nếu như bạn sử dụng [Angular\'s i18n support](http://angular.io/guide/i18n), bạn sẽ cần phải sử dụng `@angular/localize`. Đọc thêm về [$localize Global Import Migration](https://v9.angular.io/guide/migration-localize).'
+  },
+  'ng update v10': {
+    action:
+      'Thực thi lệnh `ng update @angular/core@10 @angular/cli@10` để nâng cấp Angular lên phiên bản 10.'
+  },
+  'update @angular/material': {
+    action:
+      'Thực thi lệnh `ng update @angular/material@10`.'
+  },
+  browserlist: {
+    action:
+      'Dự án mới được khởi tạo bằng Angular CLI sử dụng file tên là `.browserlistrc` thay vì `browserlist`. `ng update` sẽ thực hiện việc thay đổi này cho bạn.'
+  },
+  'v10-versions': {
+    action:
+      'Angular từ giờ sẽ yêu cầu `tslint` phiên bản 6, `tslib` phiên bản 2, và [TypeScript 3.9](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html). `ng update` sẽ thực hiện việc thay đổi này cho bạn.'
+  },
+  styleext: {
+    action: 'Dừng việc sử dụng `styleext` hoặc `spec` bên trong Angular schematics của bạn. `ng update` sẽ thực hiện việc thay đổi này cho bạn.',
```

Comment:
schematic should not be in English

Reply:
Do you have any suggestion for translating `schematic` to Vietnamese? I am not able to give a meaningful equivalent word for that.

Reply:
I would keep Angular schematics and probably having a bracket (giản đồ)? What do you think @trungk18

Reply:
I couldn't think of any blog post, articles, or any posts online that would use a literal translations of Schematics when they want to talk about Angular Schematics. I'd vote for keeping the Angular Schematics as-is (just like Angular Forms, Angular Router). What do you guys think?

Reply:
Thanks @hbweb, for the suggestion. But `(giản đồ)` is not very clear to me. Plus since we don't usually use this term in technical, it would confuse user more than help them :)

I agree with @nartc, we could keep it as original `schematics`.